src/index.ts

```diff
@@ -9,6 +9,7 @@ export async function run(): Promise<void> {
     const registry = core.getInput('registry', { required: true });
     const username = core.getInput('username', { required: true });
     const password = core.getInput('password', { required: true });
+    const tlsVerify = core.getInput('tls-verify') || 'true';
```

Comment:
Remove `|| true` because this is already set by the `default` value in `action.yml`